CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This section contains changes that have been committed but not yet released.
 
 ### Added
 
+- Calendar.isPrimary field is now available
+- Add `metadata` field in the Event model to support new event metadata
+- Add support for filtering `metadata` using `metadata_key`, `metadata_value`, and `metadata_pair`]
+
```

Comment:
Looks like a merge conflict didn't resolve quite right.  Since 1.7.0 was released, the Calendar.isPrimary line is already included there below.

Also, I'd suggest keeping a simpler one line for event metadata.  Something like:
`- Event metadata support added`

Reply:
Good catch, fixed the changelog!